thirdparty/libart/src/art.c

```diff
@@ -241,10 +291,10 @@ void* art_search(const art_tree *t, const unsigned char *key, int key_len) {
     while (n) {
         // Might be a leaf
         if (IS_LEAF(n)) {
-            n = (art_node*)LEAF_RAW(n);
+            art_leaf *leaf = LEAF_RAW(n);
```

Comment:
Are these changes related to this diff? Having these included is making this harder to read.